ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/Image/ImagePage.tsx

```diff
@@ -28,6 +28,7 @@ import EmptyStateTemplate from 'Components/PatternFly/EmptyStateTemplate';
 import { getAxiosErrorMessage } from 'utils/responseErrorUtils';
 import useURLPagination from 'hooks/useURLPagination';
 
+import HeaderLoadingSkeleton from '../../components/HeaderLoadingSkeleton';
```

Comment:
nit: This made me realize that you've factored this reusable component out to just the `Vulnerabilities/` container folder, instead of the top-level `Components/`folder.

I'll defer to your judgement for now, but I recommend that if this component would be useful in other sections of the app, we move it sooner rather than later. (You could do it in this PR, or merge and iterate in a follow-up PR.)